tools/sdk-testgen/packages/autorest.gotest/src/template/scenarioTest.go.njk

```diff
@@ -3,8 +3,22 @@
 
 {{copyright}}
 
-package {{packageName}}_scenario
-
+package {{packageName}}_test
```

Comment:
Want to know why need to change package name here?
Since the mock test and scenario test are located in different folder, will it be a problem that the scenario test and mock test use the same package name?

Reply:
I want to simplify the packages into two: one is rp_package, another is rp_package_test. For now, the scenario test's implementation is behind latest oav version. Once the mock test part is stable, I'll try to adjust the code to leverage the latest oav and test record framework.